enamel/tests/unit/api/test_version.py

```diff
@@ -11,76 +11,93 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import fixtures
+
 from enamel.tests.unit import base
 
 from enamel.api import version
 
 
-class TestVersion(base.TestCase):
+MOCK_VERSIONS = [
+    '0.1',
+    '0.9',
+    '1.0',
+]
+
+
+class MockVersionedTest(base.TestCase):
+
+    def setUp(self):
+        self.useFixture(fixtures.MonkeyPatch(
+            'enamel.api.version.VERSIONS', MOCK_VERSIONS))
+        super(MockVersionedTest, self).setUp()
+
+
+class TestVersion(MockVersionedTest):
 
     def test_version_parses(self):
-        request_version = version.Version.parse_version_string('10.5')
+        request_version = version.parse_version_string('10.5')
         self.assertEqual(10, request_version.major)
         self.assertEqual(5, request_version.minor)
 
     def test_version_negative(self):
-        request_version = version.Version.parse_version_string('-10.5')
+        request_version = version.parse_version_string('-10.5')
```

Comment:
There's no expectation that a negative version will ever be used, but a negative number is still a number, so we parse it correctly. It will never be `in_window` though, so will always cause a 406 (once that is in place, currently it's a 500).